src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1100,7 +1100,8 @@ public function column_title( $post ) {
 		$pad = str_repeat( '&#8212; ', $this->current_level );
 		echo '<strong>';
 
-		$title = _draft_or_post_title();
+		$title = $post->post_title;
+		$title = empty( $title ) ? __( '(no title)' ) : strip_tags( $title );
```

Comment:
```suggestion
		$title = empty( $title ) ? __( '(no title)' ) : wp_strip_all_tags( $title );
```

This covers the contents of `<script>` and `<style>` tags too, whereas the PHP function does not.